Source/Particles/LaserParticleContainer.cpp

```diff
@@ -192,7 +201,9 @@ LaserParticleContainer::LaserParticleContainer (AmrCore* amr_core, int ispecies,
         // Sanity checks
         int dir = WarpX::moving_window_dir;
         std::vector<Real> windir(3, 0.0);
-#if (AMREX_SPACEDIM==2)
+#if (AMREX_SPACEDIM==1)
+        windir[2+dir] = 1.0;
```

Comment:
The code is slightly misleading here.
Could instead you use `amrex::ignore_unused` for `dir` and then simply do
```
windir[2] = 1.0;
```

Reply:
Agreed. Modified it.